docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,10 +367,18 @@ which is a CDI bean and can be ``@Inject``ed.
 
 In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
 
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+[IMPORTANT]
+====
+Using a database to store transaction logs has several benefits, but in comparison with using the file system to store the logs, you might notice a reduction in performance.
 
-Quarkus allows the following JDBC-specific configuration of the object store included in `quarkus.transaction-manager.object-store.<property>` properties, where <property> can be:
+In addition, this configuration is only relevant for JTA transactions.
```

Comment:
Thanks for replacing XA with JTA. Re-reading this, I was wondering if we should use `Jakarta Transactions transactions`

Reply:
This is valid for all JTA occurrences

Reply:
The very first paragraph in [this guide](https://quarkus.io/guides/transaction) should say that the extension supports Jakarta Transactions transactions style of transaction. And thereafter the guide should only use the term transaction. In this way you can avoid the repetitive and clunky use of the phrase `Jakarta Transactions transaction`.

Reply:
+1